Stripe/STPPaymentCardTextField.m

```diff
@@ -443,9 +600,9 @@ - (void)setCardParams:(STPCardParams *)cardParams {
         [self setText:@"" inField:STPCardFieldTypeExpiration];
```

Comment:
yeah you're right. i didn't realize STPCardParams had all of those non-card related fields.